packages/eslint-config/docs/README.md

```diff
@@ -33,8 +33,6 @@ Assuming it's a typescript app, add an `extends` line to your `.eslintrc.json`:
 }
 ```
 
-That's it! You can override the standard configuration after that.
```

Comment:
is it worth adding any additional documentation to this readme that is prettier-specific? Perhaps even a link to the prettier RFC?

Reply:
@that-jill, given that it's a public repository, I'm considering not linking the private RFC. Thoughts?

Reply:
makes sense to me. thanks for considering documentation!